auto_post_classifier/api.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import FastAPI
+from auto_post_classifier.models import TaskBase
+import auto_post_classifier.utils as utils
+import json
+from json import JSONDecodeError
+from loguru import logger
+
+from pydantic import BaseModel
+
+class Post(BaseModel):
+    text: str
+    content_url: str
+
+
+iter_num = 2 
+app = FastAPI()
+
+@app.post("/rank")
+def read_root(post: Post):
+    logger.info(f"going the parse the following text:\n {post.text}")
```

Comment:
@nilnil47 adding the posts' body can make it very hard to troubleshoot if/when we'd need to review the logs. 

A quick solution that I can think of is to compute a hash of the post, emit it as part of the log's message and include the same hash in the output. This way we can link things together.

In the meantime, I think it might be better to remove the posts' body and keep my suggestion (or any other solution) to a later iteration.